src/templates/footer.hbs

```diff
@@ -71,3 +71,5 @@
     </div>
   </div>
 </footer>
+
+<script type="text/javascript" id="hs-script-loader" async defer src="//js.hs-scripts.com/21266834.js"></script>
```

Comment:
I think it would be helpful to have a comment about this being the HubSpot chat and so it is easy to grep if needed.